### PR TITLE
Added Author model to the api

### LINKED ISSUE #13: Adding Author Model to the API
Add Author model and use its reference in Quotes Model.

### PR DESCRIPTION
**Pull request by karelumair**
closes #13 #14

### CODE REVIEW

File: database/models.py [hunk 1]
```diff
@@ -1,10 +1,30 @@
 from .db import db
 import datetime
+from utils.utils import objectToJson
+
+class Author(db.Document):
+    name = db.StringField(required=True)
+    DOB = db.DateTimeField(required=True)
```
Comment (pratikGhodke1) [inline database/models.py:7]: keep column names in snake case whenever possible

File: database/models.py [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+
+    def to_json(self):
+        obj = self.to_mongo()
+        return objectToJson(obj)
+
 
 class Quote(db.Document):
     quote = db.StringField(required=True)
-    quoteBy = db.StringField(required=True)
+    quoteBy = db.ReferenceField(Author, reverse_delete_rule=db.CASCADE, dbref=False)
```
Comment (pratikGhodke1) [inline database/models.py]: Change quoteBy to author which is more descriptive

File: database/models.py (hunk 1 above)
Comment (karelumair) [reply database/models.py:7]: like `author_name`??

File: database/models.py (hunk 2 above)
Comment (karelumair) [reply database/models.py]: ok

File: database/models.py (hunk 1 above)
Comment (pratikGhodke1) [reply database/models.py:7]: Yes. Here, DOB can be replaced with dob, birth_date.

**Review by pratikGhodke1 — APPROVED** [review]

**Pull request merged**